### PR TITLE
Convert event bindings into jsactions

### PR DESCRIPTION
**Pull request by straversi**
Convert event bindings into jsactions.

TODO: this functionality should be extracted into some sort of plugin or other architectural separation.

### CODE REVIEW

File: src/lib/index.ts [hunk 1]
```diff
@@ -421,6 +420,37 @@ export class SourceFileConverter {
               );
               partTypes.push('attribute');
               continue;
+            } else if (name.startsWith('@')) {
+              // Convert event binding into a jsaction.
```
Comment (justinfagnani) [inline src/lib/index.ts]: Let's pull this out into a separate function now, even if it's in the same module. Just to reduce the size of this function.

File: src/lib/index.ts [hunk 2]
```diff
@@ -421,6 +420,37 @@ export class SourceFileConverter {
...
+              // TODO: this functionality should be extracted into some sort of
+              // plugin or other architectural separation.
+              const handlerExpression = expressions[partTypes.length];
```
Comment (justinfagnani) [inline src/lib/index.ts]: I think this is where you're going to need to add a part type for property expressions. If you have a property expression before an event expression, they'll get misaligned I think.

Comment (justinfagnani) [inline src/lib/index.ts]: Let's standardize on "listener" over "handler"

Comment (straversi) [reply src/lib/index.ts]: Did you foresee a reason for tracking the part type here?

**Pull request merged**